spec/lib/dor/assembly/stub_content_metadata_parser_spec.rb

```diff
@@ -47,5 +69,12 @@
         end
       end
     end
+
+    context 'when convert_stub_content_metadata method is called' do
+      it 'generates stub content metadata' do
```

Comment:
Another way this can be written is to use the subject.  See http://www.betterspecs.org/#subject

```suggestion
      subject { item.convert_stub_content_metadata }
      it { is_expected.to be_equivalent_to xml_doc }
```